src/pages/settings/Payments/AddDebitCardPage.js

```diff
@@ -184,6 +186,36 @@ class DebitCardPage extends Component {
         }));
     }
 
+    /**
+     * @param {String} inputExpiryDate
+     */
+    addOrRemoveSlashToExpiryDate(inputExpiryDate) {
+        this.setState((prevState) => {
+            let expiryDate = inputExpiryDate;
+
+            // Backspace was hit/character removed
+            // Remove extra digit when there's a / involved
```

Comment:
Could you just reword this sightly? It took me a couple of times to understand it properly

Reply:
I've updated. Let me know if it is fine.